addons/search-profile/style.css

```
@@ -1,3 +1,20 @@
+.tabs button .tab-icon.sa-search-profile-icon {
+  width: 24px;
+  height: 24px;
+  border-radius: 3px;
+}
+
+.tabs button.sa-search-profile-btn {
+  /* Vanilla style is width:8.75em */
+  width: auto;
+  min-width: 8.75em;
+}
+
+.sa-search-profile-btn span {
+  display: flex;
+  justify-content: center;
+}
```

This causes a layout shift when the username is shown. I think it would be better to truncate the text.

```suggestion
.sa-search-profile-btn span {
  display: block;
  overflow: hidden;
  text-overflow: ellipsis;
  text-align: center;
}
```